src/Microsoft.ML.StandardLearners/StandardLearnersCatalog.cs

```diff
@@ -256,10 +256,10 @@ public static class StandardLearnersCatalog
             string labelColumn = DefaultColumnNames.Label,
             string featureColumn = DefaultColumnNames.Features,
             IClassificationLoss lossFunction = null,
-            float learningRate = AveragedLinearArguments.AveragedDefaultArgs.LearningRate,
-            bool decreaseLearningRate = AveragedLinearArguments.AveragedDefaultArgs.DecreaseLearningRate,
-            float l2RegularizerWeight = AveragedLinearArguments.AveragedDefaultArgs.L2RegularizerWeight,
-            int numIterations = AveragedLinearArguments.AveragedDefaultArgs.NumIterations)
+            float learningRate = AveragedLinearOptions.AveragedDefaultArgs.LearningRate,
+            bool decreaseLearningRate = AveragedLinearOptions.AveragedDefaultArgs.DecreaseLearningRate,
```

Comment:
>AveragedDefaultArgs [](start = 62, length = 19)

Would rename to AveragedDefaults same for base class for this class. Just remove Arg.